config/buses/natss/100-natss.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+data:
+  gnatsd.conf: |
+    # configuration file used to override default NATS server settings
+  stan.conf: |
+    # content of configuration file used to override default NATS Streaming server settings
+kind: ConfigMap
```

Comment:
bit of a nit, could you organize this:
apiversion
kind
metadata
data

makes it easier to parse mentally :)